client/src/commands/addConnectionCommand.ts

```diff
@@ -54,29 +55,73 @@ export async function addConnection(): Promise<{} | void> {
         fabricConnectionEntry.connectionProfilePath = connectionProfilePath;
         await fabricConnectionRegistry.update(fabricConnectionEntry);
 
-        // Get the certificate file path
-        const certificatePath: string = await UserInputUtil.browseEdit('Enter a file path to the certificate file', connectionName);
-        if (!certificatePath) {
+        // Ask the user whether they want to provide a wallet or certficate and privateKey file paths
+        let wallet: IFabricWallet;
+        const answer: string = await UserInputUtil.showAddIdentityOptionsQuickPick('Chose a method for importing identity to connect with:');
+        if (!answer) {
+            // User cancelled, so do nothing
             return Promise.resolve();
-        }
+        } else if (answer === UserInputUtil.CERT_KEY) {
 
-        ParsedCertificate.validPEM(certificatePath, 'certificate');
+            // Ask for an identity name
+            const identityName: string = await UserInputUtil.showInputBox('Provide a name for the identity');
```

Comment:
is it possible to put the two paths in a separate function to make it easier to read?